base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -265,6 +265,23 @@
 				</macros>
 			</value>
 		</entry>
+		<entry>
+			<key>Keithley 2700</key>
+			<value>
+				<categories>
+					<category>Multimeters</category>
+				</categories>
+				<type>KHLY2700</type>
+				<path>Keithley/Keithley2700.opi</path>
```

Comment:
This points at an OPI which doesn't exist.

Reply:
Modified link